include/deal.II/distributed/fully_distributed_tria.h

```diff
@@ -0,0 +1,401 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2008 - 2019 by the deal.II authors
```

Comment:
Is there substantial old code that you re-used in this file? If not, please set the copyright year to "2019". It will later get expanded by some script into e.g. "2019 - 2020" if there is a change in 2020, but it cannot be done backwards.

Reply:
I did not know about this. I thought all the headers are the same.

Reply:
Then, we would need `1998 - ...` everywhere. :wink: